src/app/_components/IconButton.tsx

```diff
@@ -1,9 +1,9 @@
 import { Button } from "./Button"
 import { Icon } from "./Icon"
 
-export const IconButton = ({ class: className = "", icon, ...props }) => {
+export const IconButton = ({ class: className = "", icon, disabled = false, ...props }) => {
```

Comment:
I think you don't  need to add disabled here, because <Button spreads all props into <button and that has `disabled` prop

Reply:
It's necessary for styling (`text-neutral-${disabled ? 5 : 11} ${className}`)

Reply:
oh, you're right.